app/templates/posts.hbs

```diff
@@ -24,8 +24,8 @@
                                         {{#if post.page}}
                                             <span class="page">Page</span>
                                         {{else}}
-                                            <time datetime="{{post.publishedAtOffset}}" class="date published">
-                                                Published {{gh-format-timeago post.publishedAt}}
+                                            <time datetime="{{post.publishedAt}}" class="date published">
+                                                Published {{gh-format-timeago post.publishedAt}} ago
```

Comment:
I don't think adding "ago" here is the correct fix?
The word wasn't there before, and according to the [moment docs](http://momentjs.com/docs/#/displaying/fromnow/), it should be being output by `fromNow()`. This bug is super weird, it looks like there is something going on which may be affecting moment's locale strings?